### PR TITLE
Normative: Use method instead of value for the method

### PR DESCRIPTION
**Pull request by littledan**
This matches many people's intuition; value makes many people
think more of fields than methods.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -881,6 +892,42 @@ <h1>ToElementDescriptors ( _elementObjects_ )</h1>
...
+        1. Let _desc_ be a new Property Descriptor that initially has no fields.
+        1. Let _hasEnumerable_ be ? HasProperty(_Obj_, `"enumerable"`).
+        1. If _hasEnumerable_ is *true*, then
+          1. Let _enumerable_ be ToBoolean(? Get(_Obj_, `"enumerable"`)).
+          1. Set _desc_.[[Enumerable]] to _enumerable_.
+        1. Let _hasConfigurable_ be ? HasProperty(_Obj_, `"configurable"`).
+        1. If _hasConfigurable_ is *true*, then
+          1. Let _configurable_ be ToBoolean(? Get(_Obj_, `"configurable"`)).
+          1. Set _desc_.[[Configurable]] to _configurable_.
+        1. Let _hasValue_ be ? HasProperty(_Obj_, `"value"`).
```
Comment (nicolo-ribaudo) [inline spec.html]: "method"?

File: spec.html [hunk 2]
```diff
@@ -881,6 +892,42 @@ <h1>ToElementDescriptors ( _elementObjects_ )</h1>
...
+        1. If _hasValue_ is *true*, then
+          1. Let _value_ be ? Get(_Obj_, `"method"`).
```
Comment (nicolo-ribaudo) [inline spec.html:909]: What do you think about throwing if it isn't callable, like we do for get and set?

**Comment by pabloalmunia** [comment]
Congratulation, it's a very clear structure.

File: spec.html (hunk 2 above)
Comment (littledan) [reply spec.html:909]: Oh yeah, I meant to include that and forgot, fixed.

File: spec.html (hunk 1 above)
Comment (littledan) [reply spec.html]: Fixed.

**Comment by littledan** [comment]
Merging this patch won't mean we decided "yes" definitively; we'll still discuss it in the next decorators meeting and at TC39. Comments still welcome.

**Pull request merged**